frontend/sw360-portlet/src/main/java/org/eclipse/sw360/portal/portlets/moderation/ModerationPortletUtils.java

```diff
@@ -105,12 +106,14 @@ public static AddDocumentRequestSummary updateClearingRequest(PortletRequest req
                 ModerationService.Iface client = new ThriftClients().makeModerationClient();
                 ClearingRequest clearingRequest = client.getClearingRequestByIdForEdit(id, user);
 
-                String requestingUser = request.getParameter(ClearingRequest._Fields.REQUESTING_USER.toString());
-                if (CommonUtils.isNullEmptyOrWhitespace(requestingUser)) {
-                    log.warn("Invalid requesting user email: " + requestingUser + " is entered, by user: "+ user.getEmail());
-                    return requestSummary.setMessage("Invalid requesting user email");
-                }
-                clearingRequest.setRequestingUser(requestingUser);
+                if (isAdmin.equalsIgnoreCase("Yes")) {
```

Comment:
please use `if (PermissionUtils.isUserAtLeast(UserGroup.SW360_ADMIN, user))` instead.
Fix the indentation/formatting at line `116`.